R/to-image.R

```diff
@@ -39,156 +36,53 @@
 #' \dontrun{
 #'   # call any of these functions using either a vegaspec or a vegawidget
 #'   vw_to_svg(vegawidget(spec_mtcars))
-#'   write_png(spec_mtcars, "temp.png")
-#'   spec_mtcars %>% vw_to_png() %>% vw_png_bin()
+#'   vw_to_bitmap(spec_mtcars)
+#'   vw_write_png(spec_mtcars, "temp.png")
+#'   vw_write_svg(spec_mtcars, "temp.svg")
 #' }
-#' @seealso [webdriver::install_phantomjs()],
-#' [vega-view library](https://github.com/vega/vega/tree/master/packages/vega-view#image-export),
-#' [vw_png_bin()]
+#' @seealso [vega-view library](https://github.com/vega/vega-view#image-export),
 #'
 
 #' @rdname image
 #' @export
 #'
-vw_to_svg <- function(...) {
-  UseMethod("vw_to_svg")
-}
+vw_to_svg <- function(spec, seed = sample(1e8, size = 1), base_url = "", width = NULL, height = NULL) {
```

Comment:
Would you consider an alternate ordering of the arguments? I think width and height may be more commonly used (but I could be wrong):

```r
vw_to_svg <- function(spec, width = NULL, height = NULL, base_url = "", seed = sample(1e8, size = 1))
```